spec/index.bs

```diff
@@ -605,7 +603,7 @@ interface GPUProgrammablePassEncoder : GPUObjectBase {
     void insertDebugMarker(DOMString markerLabel);
 };
 
-interface GPURenderPassEncoder : GPUProgrammablePassEncoder {
+interface GPURenderEncoderBase : GPUProgrammablePassEncoder {
```

Comment:
setting viewports and scissors is not supported by D3D12 bundles

Reply:
Bundles are supposed to be relatively small, so they could be split at `setScissor` or `setViewport` boundary.

Reply:
Did you consider having the user to split them in this case? In my mind, viewport and scissor are associated more with the render targets than with current draws, so I don't think it would harm bundle semantics too much if we move those out